### PR TITLE
Update validator page

### PR DESCRIPTION
**Pull request by TopETH**

### CODE REVIEW

**Comment by cuteolaf** [comment]
- make fonts smaller for boxes
- clip the owner address with '...' when overflow
- hover effect on each box
- when clicked & selected, performance chart of the selected subnet should be shown

File: src/screens/validator.tsx [hunk 1]
```diff
@@ -84,6 +92,76 @@ const validatorDescription = css`
...
+	border: 1px solid gray;
+	padding: 5px;
+	cursor: pointer;
+
+	&:hover {
+		border-color: white;
+	}
+`;
+const selectedNeuronBox = css`
+	border-color: #7aff97 !important;
```
Comment (cuteolaf) [inline src/screens/validator.tsx:131]:
```suggestion
	border-color: #14dec2 !important;
```

File: src/components/subnets/ColdkeySubnets.tsx [hunk 2]
```diff
@@ -26,6 +26,7 @@ export const ColdkeySubnets = ({ netUid, coldkey }: ColdkeySubnetsProps) => {
 			netUid: { equalTo: netUid },
 			coldkey: { equalTo: coldkey },
 		},
+		1024,
```
Comment (cuteolaf) [inline src/components/subnets/ColdkeySubnets.tsx:29]: we can't remove this constant?

**Review by cuteolaf — CHANGES_REQUESTED** [review]

File: src/screens/validator.tsx [hunk 3]
```diff
@@ -195,12 +267,12 @@ export const ValidatorPage = () => {
 	const subnets = useSubnets(undefined);
 
 	const [activeSubnet, setActiveSubnet] = useState(-1);
-	const subnetIDs = useMemo(() => {
-		const ids: number[] = validator.data?.validatorPermits || [];
+	useEffect(() => {
+		const ids: number[] =
+			neuronMetagraph.data?.map((meta) => meta.netUid) || [];
```
[the remark this answers is not in the document]
Comment (TopETH) [reply src/screens/validator.tsx]: Removed.

**Review by cuteolaf — APPROVED** [review]

**Comment by cuteolaf** [comment]
create a new PR to remove the constants.

**Pull request merged**